exercises/secret-handshake/src/main/java/Instruction.java

```diff
@@ -0,0 +1,5 @@
+enum Instruction {
+
+    WINK, DOUBLE_BLINK, CLOSE_YOUR_EYES, JUMP, REVERSE
```

Comment:
Maybe this should be an `Action` type and `REVERSE` should not be a part of it? 🤔